pygmt/src/contour.py

```diff
@@ -36,13 +36,13 @@
 )
 @kwargs_to_strings(R="sequence", c="sequence_comma", i="sequence_comma", p="sequence")
 def contour(self, x=None, y=None, z=None, data=None, **kwargs):
-    """
+    r"""
     Contour table data by direct triangulation.
 
     Takes a matrix, (x,y,z) pairs, or a file name as input and plots lines,
     polygons, or symbols at those locations on a map.
 
-    Must provide either *data* or *x*, *y*, and *z*.
+    Must provide either **data** or **x**/**y**/**z**.
 
     [TODO: Insert more documentation]
```

Comment:
Are you planning on leaving this TODO in?

Reply:
I'll fix with [issue 879](https://github.com/GenericMappingTools/pygmt/issues/879) in a separate PR.

Reply:
Fixed in #883